wildlifelicensing/apps/applications/pdf.py

```diff
@@ -0,0 +1,182 @@
+    # dates and licensing officer
+    metadata_table_style = TableStyle([('VALIGN', (0, 0), (-1, -1), 'TOP')])
+
+    licensee = application.applicant.get_full_name()
+    if application.lodgement_number and application.lodgement_sequence:
+        lodgement_number = '{}-{}'.format(application.lodgement_number, application.lodgement_sequence)
+    else:
+        lodgement_number = ''
+
+    top_row = [
+        Paragraph('Licensee:', styles['BoldLeft']),
+        Paragraph(licensee, styles['Left']),
+        Paragraph('Lodgement Number:', styles['BoldLeft']),
+        Paragraph(lodgement_number, styles['Left'])
+    ]
+
+    address = application.applicant_profile.postal_address
+    address_paragraphs = [Paragraph(address.line1, styles['Left']), Paragraph(address.line2, styles['Left']),
+                          Paragraph(address.line3, styles['Left']),
+                          Paragraph('%s %s %s' % (address.locality, address.state, address.postcode), styles['Left']),
+                          Paragraph(address.country.name, styles['Left'])]
+
+    lodgement_date = application.lodgement_date.strftime(DATE_FORMAT) if application.lodgement_date is not None else ''
+
+    bottow_row = [
+        Paragraph('Address:', styles['BoldLeft']),
+        address_paragraphs,
+        Paragraph('Lodgement Date:', styles['BoldLeft']),
+        Paragraph(lodgement_date, styles['Left'])
+    ]
+
+    left_heading_cell_width = 60
+    right_heading_cell_width = 110
+    value_cell_width = 140
+
+    return Table([top_row, bottow_row],
+                 colWidths=[left_heading_cell_width, value_cell_width, right_heading_cell_width, value_cell_width],
+                 style=metadata_table_style)
+
+
+def _format_field_value(field, value):
+    if field['type'] in ['radiobuttons', 'select']:
+        return value.title() if value else MISSING_VALUE_PLACEHOLDER
+    elif field['type'] == 'declaration':
+        return 'Declaration checked' if value == 'on' else 'Declaration not checked'
+    elif field['type'] == 'checkbox':
+        return 'Yes' if value == 'on' else 'No'
+    elif field['type'] == 'file':
+        return value if value else MISSING_FILE_PLACEHOLDER
+    else:
+        return value if value else MISSING_VALUE_PLACEHOLDER
+
+
+def _create_application_questionaire(field, data, elements, indent_index):
+    paragraph_style = ParagraphStyle('', spaceAfter=PARAGRAPH_BOTTOM_MARGIN, leftIndent=indent_index * PARAGRAPH_INDENT)
+    if field['type'] == 'section':
+        paragraph_style.fontName = BOLD_FONTNAME
+        paragraph_style.fontSize = LARGE_FONTSIZE
+        elements.append(Paragraph('<u>{}</u>'.format(field['label']), paragraph_style))
+
+        elements.append(Spacer(1, SECTION_BUFFER_HEIGHT))
+
+        for child_data in data[field['name']]:
+            for child_field in field['children']:
+                _create_application_questionaire(child_field, child_data, elements, indent_index)
```

Comment:
I'm not sure but there's an `indent_index + 1` on other recursions below and not here. Voluntary?